app/modules/tasks/components/task_breadcrumb.svelte

```diff
@@ -0,0 +1,94 @@
+<script>
+  import { without } from 'underscore'
+  import Link from '#app/lib/components/link.svelte'
+  import { icon } from '#app/lib/icons'
+  import Dropdown from '#components/dropdown.svelte'
+  import { entitiesTypesByTypes } from '#tasks/components/lib/tasks_helpers'
+  import { i18n } from '#user/lib/i18n'
+
+  export let entitiesType, type, taskId
+  $: menuEntitiesTypes = without(entitiesTypesByTypes[type], entitiesType)
+  $: menuTypes = without([ 'merge', 'delete', 'deduplicate' ], type)
+</script>
+<div class="breadcrumb">
+  <Link
+    url="/tasks"
+    text={i18n('dashboard')}
+    classNames="link"
+  />
+  <span class="separator">/</span>
+  <Dropdown dropdownWidthBaseInEm={10}>
+    <div slot="button-inner">
+      {i18n(type)}
```

Comment:
Capitalized strings should be dedicated to sentences only. Some exceptions are possible, but not for this

Reply:
Since when? 
![2024-11-16_23-09](https://github.com/user-attachments/assets/fcf8917a-9300-4a80-af31-bdbd59cc4617)

Reply:
We mentioned orally this convention a while ago (but that doesn't mean I will spend some time on un-capitalizing legacy code). Since this comment is non-blocking, I thought it would be okay for you to leave it that way. [Capitalization is for sentences](https://www.grammarly.com/blog/punctuation-capitalization/capitalization-rules/)

Reply:
"Task" got capitalized in a2ca7727e53ba as it could be considered the first term of the breadcrumb sequence